components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -16,6 +16,8 @@
 #import "MDCTabBarViewDelegate.h"
 #import "private/MDCTabBarViewItemView.h"
 
+#import <CoreGraphics/CoreGraphics.h>
```

Comment:
Why do we need to import CoreGraphics here?

Reply:
for CGRect things.

Reply:
Which CGRect things? I'm not seeing any CGRect manipulation in this PR

Reply:
For example, `layoutSubviews` uses `CGFloat` and `CGRectGetWidth`. This PR did not add those calls but I noticed the missing import statement while writing this one.